mass_run.sh

```diff
@@ -0,0 +1,14 @@
+declare -a listOfFilesToRun=("*desnat_straight_once*" "*floor_2_obright*" "*desnat_straight_three*" "*209-to-occam*" "*robolab_straight_on*" "*floor_2_obleft*" "*floor_2_right_once*" "straight_209_occam" "*209_occam_obleft_once*" "*mac-1-2-official*")
```

Comment:
This file needs some documentation.  It runs a bunch of stuff.  How was the list selected?  What / where would one find the outputs?